sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/clienttelemetry/ClientTelemetryMetrics.java

```diff
@@ -1098,6 +1123,26 @@ private void recordGatewayStatistics(
                     }
                 }
 
+                if (actualItemCount > 0) {
```

Comment:
One thing I learned recently is that Prometheus will cause issues when you sometimes add a tag and sometimes not. So, if this tag is enabled (like if actualItemCount>1 we would publish it) we should always publish it - use 0 when necessary.

Reply:
removed the check